examples/workers/ColdStorageWorker.js

```diff
@@ -22,7 +22,7 @@ class ColdStorageWorker extends Worker {
     const { walletConsolidator } = this;
     const utxos = this.getUTXOS();
     if (utxos.length === 0) {
-      logger.error('ColdStorageWorker : We did not found any utxos. Doing nothing');
+      throw new Error('ColdStorageWorker : We did not found any utxos. Doing nothing');
```

Comment:
Don't we want to introduce specific error classes?

Reply:
It's actually not an error to not find a UTXO afaik, that's why it just tell you about it and continue process.